controllers/src/test/scala/cromwell/pipeline/controller/UserControllerTest.scala

```diff
@@ -19,23 +21,23 @@ class UserControllerTest
     with Matchers
     with ScalatestRouteTest
     with MockitoSugar
-    with BeforeAndAfterAll {
-  import PlayJsonSupport._
+    with PlayJsonSupport {
```

Comment:
We discussed that we should import `PlayJsonSupport._` instead of mixing it. Why did you change it here?

Reply:
Done